### DISCUSSION TITLE
How to host server

### DISCUSSION THREAD

**Discussion by brianbaldner** [scottoffen/grapevine#129] [Q&A] [answered]
What would be the best way to host a server. I was looking at Heroku but it didn't look like it worked with c#. I would like to know what you guys use.

**Comment by scottoffen** [comment]
I'm not sure about the broader community, but I don't use Grapevine to host public-facing servers. For public facing servers I use ASP.NET on either Azure or AWS.

**Reply by brianbaldner** [reply]
Thanks for the reply

**Comment by brianbaldner** [comment, marked as answer]
Yeah, I figured that out. I avoided using asp.net because I thought their template was really confusing, but they have a new template that feels a lot like grapevine, and I ported my code in like 10 min and hosted it for free on Heroku. If anyone else is in that situation I recommend you do the same.

**Reply by scottoffen** [reply]
It's likely not an accident that the new template feels like Grapevine...
